content/blog/getting-started-with-the-hpe-greenlake-cloud-eventing-framework.md

```diff
@@ -0,0 +1,306 @@
+
+![Post event in Postman](/img/post-event-in-postman.jpg "Post event in Postman")
+
+Click **Send** in Postman, and check in Make (in the scenario’s history) that the second route was taken, as shown below.
+
+![Second route was taken](/img/second-route-taken.jpg "Second route was taken")
+
+You can also verify that in the **Google sheets** module, the content of the payload was mapped to different values (A:, B:, C:,…)
+
+![Google sheet property mapping ](/img/googlesheet-set.jpg "Google sheet property mapping ")
+
+Finally verify that your Google sheet was updated and a new row was added using the payload content, as shown in my Google sheet below.
+
+![Row was added in Google sheet](/img/google-row-visible.jpg "Row was added in Google sheet")
+
+## Putting it all together
+
+It’s now time to put it all together. Connect to HPE GreenLake cloud console and select to **Manage Workspace**. From there, select the **Automations** tile and finally the **Webhooks** tile to access the webhooks configuration section:
+
+![Webhooks in HPE GreenLake cloud](/img/webhooksview.jpg "Webhooks in HPE GreenLake cloud")
+
+To register a new webhook as shown in the capture above, you need to provide:
+
+* A name
+* A description
+* The URL of the webhook
+* The Webhook secret key. (When you set this secret key don’t forget to go back to Make and edit the **Tool Set variable** module to use the same secret key. These keys MUST match exactly!)
+
+![You must use the same secret key on both ends](/img/same-secret-key.jpg "You must use the same secret key on both ends")
+
+### Register your webhook
+
+When you select **Register webhook**, the first thing HPE GreenLake cloud will do is to establish a trust relationship with the webhook pointed by the URL, by sending a challenge payload using an HTTP POST request (as already discussed in the previous section).
+
+If the webhook response is the expected one, then the webhook is placed in **Active** state. Otherwise, after a few attempts, the webhook state will be set to **Critical**. At any point in time you can use the **Test** button, to submit a new challenge (after fixing an issue with your webhook, for example).
+
+### It’s time to subscribe to events
+
+From the web console, you can now **subscribe to events** available from the HPE GreenLake services. You can find the list of those already available from the [HPE GreenLake Developer Portal](https://developer.greenlake.hpe.com/docs/greenlake/services/event/public/ui/#finding-events-on-hpe-greenlake-developer-portal). The list will grow over time as new services adopt this eventing framework.
+
+Select your webhook from the list and select Subscribe to event. Select the source Service manager (there is only HPE GreenLake Platform for now), then cut/paste the event name from the [event catalog](https://developer.greenlake.hpe.com/docs/greenlake/services/#event-catalog). For example:
```

Comment:
Should "Subscribe to event" be in a bold font here in this first sentence? You seem to be bolding it in other places.